cluster/juju/layers/kubernetes-worker/templates/kube-proxy-kubeconfig

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Config
+clusters:
+  - cluster:
+      certificate-authority: /etc/kubernetes/ca.pem
+      server: https://node0.c.kubestack.internal:6443
+    name: kubernetes
+contexts:
+  - context:
+      cluster: kubernetes
+      user: proxy
+    name: proxy-to-kubernetes
+current-context: proxy-to-kubernetes
+users:
+  - name: proxy
+    user:
+      client-certificate: /etc/kubernetes/client.pem
+      client-key: /etc/kubernetes/client-key.pem
```

Comment:
The keys are written with a .key extension, we should either fix this or change the tls code.